### PR TITLE
DOCSP-39027: Add CMAP diagram to FAQ page

### PR DESCRIPTION
**Pull request by mcmorisi**
# Pull Request Info
Adding a simple illustrative CMAP diagram to FAQ page.

[PR Reviewing Guidelines](https://github.com/mongodb/docs-golang/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-39027>
Staging - <https://preview-mongodbmcmorisi.gatsbyjs.io/csharp/DOCSP-39027/faq/>

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?

### CODE REVIEW

File: source/includes/figures/CMAP_diagram.svg
Comment (rustagir) [inline source/includes/figures/CMAP_diagram.svg:1]: S: could a different image be used for the client application and mongoclient? i feel like the mongoclient could be the person figure and the application should be the C#/.NET logo or something like that

File: source/faq.txt [hunk 1]
```diff
@@ -44,7 +44,11 @@ support concurrent MongoDB operations in your multi-threaded application.
 The maximum size of each connection pool is set by the ``MaxConnectionPoolSize`` option, which
 defaults to ``100``. If the number of in-use connections to a server reaches
 the value of ``MaxConnectionPoolSize``, the next request to that server will wait
-until a connection becomes available.
+until a connection becomes available. The following diagram illustrates a high-level view
+of the ``MongoClient``'s connection pool functionality:
```
Comment (rustagir) [inline source/faq.txt:48]:
```suggestion
until a connection becomes available. The following diagram illustrates a high-level view
of how the ``MongoClient`` manages a connection pool:
```

**Review by rustagir — CHANGES_REQUESTED** [review]
Happy to discuss my comments or collaborate on the image before the next official review!

File: source/includes/figures/CMAP_diagram.svg
Comment (rustagir) [inline source/includes/figures/CMAP_diagram.svg:1]: S: could you also add some text between mongoclient and connection pool?

**Review by rustagir — APPROVED** [review]
left another suggested improvement but lgtm!

File: source/includes/figures/CMAP_diagram.svg
Comment (mcmorisi) [reply source/includes/figures/CMAP_diagram.svg:1]: Will do!

**Pull request merged**